### PR TITLE
PEP8ifying and cleaning up some non-pythonic things

### PR DESCRIPTION
**Pull request by alexk307**

### CODE REVIEW

File: http.py [hunk 1]
```diff
@@ -6,7 +6,6 @@
 import time
 import StringIO
 import gzip
-import sys
```
Comment (alexk307) [inline http.py]: Not used

File: util.py [hunk 2]
```diff
@@ -1,17 +1,19 @@
 import glob
 import yaml
-import syslog
```
Comment (alexk307) [inline util.py]: not used

File: http.py [hunk 3]
```diff
@@ -114,8 +115,9 @@ def __init__(self, http_request):
         self.response_object = None
         self.request = None
         self.sock = None
-        self.CIPHERS = 'ADH-AES256-SHA:ECDHE-ECDSA-AES128-GCM-SHA256: \
-                ECDHE-RSA-AES128-GCM-SHA256:AES128-GCM-SHA256:AES128-SHA256:HIGH:'
+        self.CIPHERS = \
```
Comment (zmallen) [inline http.py]: Are these two strings equivalent, aka the old one and the new one? Can you run a quick test in shell and confirm? We may get undefined behavior if spaces are mismatched

File: http.py [hunk 4]
```diff
@@ -131,7 +133,8 @@ def send_request(self):
             self.sock.send(self.request)
         except socket.error as exc:
             print exc
-        self.get_response()
+        finally:
```
Comment (zmallen) [inline http.py]: Can you move this after `self.sock.send(self.request)? Dont want to call recv on a socket that errors out

Comment (alexk307) [inline http.py]: thats how it was before since the exc was just being printed, it was being called anyways

Comment (zmallen) [inline http.py]: whoa, thats not right. @csanders-git do you think self.get_response should be after `self.sock.send(self.request)`? Then 135 should have a TestError be thrown

File: http.py (hunk 3 above)
Comment (alexk307) [inline http.py]: The way it was before had the \ inside of the string which looked like

```
>>> self.CIPHERS
'ADH-AES256-SHA:ECDHE-ECDSA-AES128-GCM-SHA256:                 ECDHE-RSA-AES128-GCM-SHA256:AES128-GCM-SHA256:AES128-SHA256:HIGH:'
```

when evaluated which I assume is not correct? It's all supposed to be one string right?

Comment (zmallen) [inline http.py]: @csanders-git thoughts?

**Comment by zmallen** [comment]
@alexk307 can you fix the conflicts and rebase?

**Comment by alexk307** [comment]
@zmallen Rebased

File: testrunner.py [hunk 5]
```diff
@@ -41,3 +41,15 @@ def run_stage(self, stage, logger_obj):
         http_ua = http.HttpUA(stage.input)
         http_ua.send_request()
         self.test_status(stage.output.status, http_ua.response_object.status)
+
```
Comment (alexk307) [inline testrunner.py]: Was this supposed to be removed or what? @zmallen

Comment (zmallen) [inline testrunner.py]: This got moved down to line 55

**Comment by zmallen** [comment]
Because this is functionally equivalent to the code from before, im going ot merge this and create 2 issues related to get_response and the SSL ciphers. Thanks for the help @alexk307 !

**Pull request merged**